tests/foreman/api/test_permission.py

```diff
@@ -265,7 +260,10 @@ def set_taxonomies(self, entity, organization=None, location=None):
         return entity
 
     @tier1
-    def test_positive_check_create(self):
+    @pytest.mark.parametrize(
+        'entity', **parametrized([entities.Architecture, entities.Domain, entities.ActivationKey])
+    )
+    def test_positive_check_create(self, entity):
         """Check whether the "create_*" role has an effect.
 
         :id: e4c92365-58b7-4538-9d1b-93f3cf51fbef
```

Comment:
@peterdragun For parametrized tests you need to add `:parametrized: yes` token in the docstrings(after test id)

Reply:
Thanks, updated

Reply:
I don't see this token included in this function docblock? get missed in a commit maybe?